src/intl/messages.js

```diff
@@ -62,6 +62,13 @@ export const messages: { [messageId: string]: MessageType } = {
   cancel: 'Cancel',
   cannotUpdateCloudInitHostname: 'Host name cannot be synchronized with VM name because the VM name is not a valid host name.',
   cantLogonToConsole: 'Single sign on failed. Please check to see if the guest agent is running on your virtual machine. Contact your administrator if the problem continues.',
+  cantOpenConsole: "Can't open console: {message}",
+  cantOpenGraphicalConsoleEmptyList: "Can't open graphical console since {vmName} is a headless VM.",
+  cantOpenGraphicalConsoleMoreOptions: "Can't open graphical console you can find more options in the VM page.",
+  cantOpenGraphicalConsoleNoDefault: "Can't open graphical console {vmName} has no {defaultConsole} console assigned. You can find more options in the VM page.",
```

Comment:
Please rephrase to:
Can't open a console since VM {vmName} is not assigned with the default console {defaultConsole}....